src/duct.py

```diff
@@ -97,57 +137,52 @@ def collect_sample(self):
                         "rss": int(rss),
                         # Virtual Memory size
                         "vsz": int(vsz),
+                        "timestamp": datetime.utcnow().isoformat() + "Z",
```

Comment:
let's use localized time stamps,
```suggestion
                        "timestamp": datetime.now().astimezone().isoformat(),
```